src/ansys/systemcoupling/core/syc_version.py

```diff
@@ -25,7 +25,7 @@
 # Define constants relating to the default/current version of System Coupling
 
 SYC_MAJOR_VERSION = 24
-SYC_MINOR_VERSION = 1
+SYC_MINOR_VERSION = 2
```

Comment:
This reminded me that I had also made this comment at one point but it seemed to have disappeared. I think it was because of a force push - these seem to have a detrimental effect on reviews that are in progress. We should probably make a point of avoiding them unless there is a good reason to use one. @ochernuk @licui7

Reply:
you mean not using force push?